src/mlpack/methods/dbscan/dbscan_impl.hpp

```diff
@@ -68,25 +71,44 @@ size_t DBSCAN<RangeSearchType, PointSelectionPolicy>::Cluster(
   centroids.zeros(data.n_rows, numClusters);
 
   // Calculate number of points in each cluster.
-  arma::Row<size_t> counts;
-  counts.zeros(numClusters);
-  for (size_t i = 0; i < data.n_cols; ++i)
+  arma::Row<size_t> counts(numClusters, arma::fill::zeros);
```

Comment:
We are getting rid of `arma::fill::zeros` can you try removing it no worries if you have any specific reason for adding this.

Reply:
actually `arma::fill::zeros` is not needed, since the min version of armadillo will do zero initialization by default if we do not have anything specified

Reply:
Oh cool we can remove this for sure then :+1:

Reply:
@MarkFischinger  can you remove this? as described in the above comment ? thanks